docs/byoh_agent.md

```diff
@@ -0,0 +1,121 @@
+# BYOH Agent
+
+BYOH agent is a binary that runs on the hosts and its responsibility include -  
+1. Setup - Install kubernetes components on the host
+2. Registration - Register the host to the cluster capacity pool
+3. Bootstrap - Convert the host to a kubernetes node
+
+## Usage of BYOH agent
+
+Below flags are supported by the agent:-  
+```
+--downloadpath string 
+```
+File System path to keep the downloads (default `/var/lib/byoh/bundles`)  
+<br />
+
+```
+--feature-gates map[String]Bool
+```
+A set of key-value pairs that describe feature gates for alpha/experimental features.  
+<br />
+```
+--kubeconfig string           
```

Comment:
I don't think this is supported anymore, we can remove this from the documentation

Reply:
This flag has been renamed to bootstrap-kubeconfig, will update this